packages/core/test/WorkflowExecute.test.ts

```diff
@@ -120,6 +120,8 @@ describe('WorkflowExecute', () => {
 						if (nodeData.data === undefined) {
 							return null;
 						}
+						if (testData.output.testAllOutputs)
```

Comment:
not sure how I feel about the two different return types. could we do something like this instead, and update all the tests accordingly?

```typescript
const toMap = testData.output.testAllOutputs ? nodeData.data.main : [nodeData.data.main[0]!];
return toMap.map(data => data!.map(entry => entry.json));
```

and in the long run we should update all tests to test all outputs, and then remove the `testAllOutputs` flag.

Reply:
Ah, that's a valid point. However, this would also require us to change the `expect` line below it, because this is now an array of arrays (and some tests do have multiple runs, so it would require modifying the expected output for each run):

```
const expectedOutput = testData.output.nodeData[nodeName].map(outputData => {
    return testData.output.testAllOutputs ? outputData : [outputData];
})

// expect(resultData).toEqual(testData.output.nodeData[nodeName]);
expect( resultData).toEqual(expectedOutput);
```

Reply:
Or do you mean update all tests to include this in the tests themselves?